### PR TITLE
feat: add `playwright` example

### PR DESCRIPTION
**Pull request by irvile**
This PR adds [playwright](https://playwright.dev/) example.

### CODE REVIEW

File: playwright/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "dev": "remix dev",
+    "start": "remix-serve build",
+    "test:e2e": "playwright test"
+  },
+  "dependencies": {
+    "@remix-run/node": "*",
+    "@remix-run/react": "*",
+    "@remix-run/serve": "*",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```
Comment (MichaelDeBoey) [inline playwright/package.json:15]: Let's keep this the same as in `_template`

```suggestion
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
```

File: playwright/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "devDependencies": {
+    "@playwright/test": "^1.27.1",
+    "@remix-run/dev": "*",
+    "@remix-run/eslint-config": "*",
+    "@types/react": "^18.0.21",
+    "@types/react-dom": "^18.0.6",
```
Comment (MichaelDeBoey) [inline playwright/package.json:22]:
```suggestion
    "@types/react": "^17.0.39",
    "@types/react-dom": "^17.0.13",
```

File: playwright/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "eslint": "^8.10.0",
+    "typescript": "^4.8.4"
```
Comment (MichaelDeBoey) [inline playwright/package.json:24]:
```suggestion
    "typescript": "^4.7.4"
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: playwright/package.json (hunk 1 above)
Comment (irvile) [reply playwright/package.json]: sure! its done now.

**Review by machour — APPROVED** [review]
Nice work, thanks!

**Comment by irvile** [comment]
> Nice work, thanks!

It's a pleasure! I love remix.

**Review by MichaelDeBoey — APPROVED** [review]

**Review by mcansh — APPROVED** [review]

**Comment by mcansh** [comment]
LGTM, thanks @irvile!

**Pull request merged**